config/default/common/config/wv.json/stories/default/worldview_intro.json

```diff
@@ -108,8 +108,8 @@
                 "placementBeacon": "bottom"
               },
               {
-                "target": "#wv-link-button",
-                "content": "Create a custom URL of your Worldview page to share with friends and colleagues!",
+                "target": "#wv-share-button",
```

Comment:
We should probably write a test for this tour story that steps through each step so that we catch changes like this in the future.  Changing the ID would break that step (and the rest of the Joyride tour?)

Reply:
Luckily, the name change that occurred with the addition of embed only broke the step, not the whole tour. A test to check for these kinds of changes would be helpful.